src/modules/simulator/simulator_mavlink.cpp

```diff
@@ -73,9 +73,12 @@ const unsigned mode_flag_armed = 128;
 const unsigned mode_flag_custom = 1;
 
 using namespace simulator;
+using namespace time_literals;
 
-void Simulator::pack_actuator_message(mavlink_hil_actuator_controls_t &msg, unsigned index)
+mavlink_hil_actuator_controls_t Simulator::pack_actuator_message(const actuator_outputs_s &actuators)
```

Comment:
I think while you're at it the name `pack` is misleading, and it's already used in the mavlink C header terminology.
What about `actuator_controls_from_actuator_outputs()`?

Reply:
Updated.